src/FsAutoComplete.Core/UntypedAstUtils.fs

```diff
@@ -115,6 +116,9 @@ let internal getRangesAtPosition input (r: Position) : Range list =
       addIfInside r
       List.iter walkType ts
       walkMemberSig sign
+    | SynTypeConstraint.WhereSelfConstrained (t, r) ->
```

Comment:
This is a new F# 7 feature - self constrained types. Walking them means checking the range and walking the contained type.